<% if object.errors.any? %>
  <div id="error_explanation">
    <p>
			Oops. Please fix these errors and try again.
			<!--<%= pluralize(object.errors.count, "error") %> first.
        prohibited this <%= object.class.to_s.underscore.humanize.downcase %> 
        from being saved:-->
		</p>
    <!--<p>There were problems with the following fields:</p>-->
    <ul>
    <% object.errors.full_messages.each do |msg| %>
      <li><%= msg %></li>
    <% end %>
    </ul>
  </div>
<% end %>
